### ISSUE TITLE
Ansi colors & styles in exception handlers

### ISSUE THREAD

**Issue by charphi** [remkop/picocli#484]
Currently, when an exception arises, `DefaultExceptionHandler` prints the raw message on syserr.
Ansi colors & styles could improve the error messages; make them clearer, expecially when dealing with missing parameters.

**Comment by remkop** [comment]
I don't oppose this, but beware that in future, it should be possible to provide error messages from a resource bundle (#485). Not sure if that changes anything for this ticket, but it is something to bear in mind.

Do you think you will be able to provide a pull request for this?

**Comment by remkop** [comment]
What I mean is that the markup for ANSI colors and styles can live in the resource bundle. For example:

```
err.missing.required = Missing required option @|red,bold {0}|@
```

That way users can customize the colors if they want.

**Comment by remkop** [comment]
Fixed in #845. This will be included in picocli 4.3.
